optaplanner-examples/pom.xml

```diff
@@ -175,6 +175,11 @@
         </exclusion>
       </exclusions>
     </dependency>
+      <dependency>
+          <groupId>org.assertj</groupId>
```

Comment:
@ge0ffrey would probably like to use only junit assertions before we migrate to assertj. I don't see a value in rewriting existing test into junit for the sake of consistency and then migrating them back to assertj several weeks/months later.

Reply:
@Fonifan Please fix the indentation and move the dependency to a `<!-- Testing -->` section (you'll have to create it) between "XML" and "Examples only dependencies". It should look like
```xml

    <!-- XML -->
    <dependency>
      <groupId>com.thoughtworks.xstream</groupId>
      <artifactId>xstream</artifactId>
    </dependency>
    <!-- Testing -->
    <dependency>
      <groupId>org.assertj</groupId>
      <artifactId>assertj-core</artifactId>
      <scope>test</scope>
    </dependency>

    <!-- Examples only dependencies. -->
```

One reason is consistency with the rest of OptaPlanner modules' POMs where dependencies are organized in sections (internal, external, logging, commons, testing, ...). Another reason is that the "Examples only dependencies" section has a special purpose (see the comment at the beginning of the section), which doesn't apply to the AssertJ dependency.

Reply:
+1, thanks for noticing this.